src/ai/backend/storage/vfs/__init__.py

```diff
@@ -38,8 +40,9 @@
 
 
 async def run(cmd: str) -> str:
-    proc = await asyncio.create_subprocess_shell(
-        cmd,
+    split_cmd = shlex.split(cmd)
```

Comment:
This is not what I want. The command should be generated as string tokens with no manual parsing in the first place.